src/scripts/main.js

```diff
@@ -1,17 +1,42 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const mainElement = document.querySelector('body');
+  const block = document.createElement('div');
+
+  block.classList.add('notification');
+  block.classList.add(`${type}`);
+  block.style.top = `${posTop}px`;
+  block.style.right = `${posRight}px`;
+  mainElement.append(block);
+
+  const h2 = document.createElement('h2');
+
+  h2.classList.add('title');
+  h2.innerText = title;
+  h2.style.whiteSpace = 'nowrap';
+  block.append(h2);
+
+  const p = document.createElement('p');
+
+  p.innerText = description;
+  block.append(p);
 };
 
-pushNotification(10, 10, 'Title of Success message',
-  'Message example.\n '
+setTimeout(() => {
+  pushNotification(10, 10, 'Title of Success message',
+    'Message example.\n '
   + 'Notification should contain title and description.', 'success');
+}, 2000);
```

Comment:
all notifications should appear immediately and removed after 2s.
it is possible reason - why tests are failed